### PR TITLE
Clean up godeps after manual additions

### PR DESCRIPTION
**Pull request by jimmidyson**
Fixes up some ordering in godep manifest, missing dependency & redundant dependencies.

@mwielgus

### CODE REVIEW

**Comment by piosz** [comment]
Did you have a chance to test the build manually (with empty local gopath)?

**Comment by jimmidyson** [comment]
@piosz I relied on travis to do that for me :)

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -9,35 +9,26 @@
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
 			"Comment": "null-5",
-			"Rev": "75cd24fc2f2c2a2088577d12123ddee5f54e0675"
+			"Rev": "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
```
Comment (piosz) [inline Godeps/Godeps.json]: Why this is changed?

File: Godeps/Godeps.json [hunk 2]
```diff
@@ -9,35 +9,26 @@
...
+			"ImportPath": "github.com/bigdatadev/goryman",
+			"Rev": "55c3cbc3df542201b3c140e72a758d0adae4bc05"
+		},
+		{
 			"ImportPath": "github.com/boltdb/bolt",
 			"Comment": "v1.0-113-g980670a",
 			"Rev": "980670afcebfd86727505b3061d8667195234816"
 		},
 		{
-			"ImportPath": "github.com/coreos/go-semver/semver",
```
Comment (piosz) [inline Godeps/Godeps.json]: Not sure whether you can remove coreos deps. I tried to do it some time ago and it broke the build:(

**Comment by piosz** [comment]
LGTM. Hopefully the tests will pass.

File: Godeps/Godeps.json (hunk 2 above)
Comment (jimmidyson) [inline Godeps/Godeps.json]: Seems to work OK...

File: Godeps/Godeps.json (hunk 1 above)
Comment (jimmidyson) [inline Godeps/Godeps.json]: Must be a change in godep binary - this is cleanly generated via `godep save` so just a tooling thing I guess.

Would like to change to a different vendoring tool like glide soon though tbh. I find godep pretty annoying to work with personally & would prefer the standard golang `vendor` dir anyway.

**Comment by jimmidyson** [comment]
Not sure if `default` Jenkins build should pass? `Jenkins GCE es2` passes fine...

**Comment by piosz** [comment]
`default` failed as expected

File: Godeps/Godeps.json (hunk 1 above)
Comment (piosz) [inline Godeps/Godeps.json]: That's fine.

File: Godeps/Godeps.json (hunk 2 above)
Comment (piosz) [inline Godeps/Godeps.json]: The tests passed, so seems to be ok.

**Comment by jimmidyson** [comment]
Thanks @piosz!

**Pull request merged**